StRoot/RTS/src/TRG_FCS/fcs_trg_base.cxx

```diff
@@ -540,7 +540,7 @@ int fcs_trg_base::end_event()
 
 	verify_event_io() ;	// verify interconnectivity 
 
-	//int dsmout = 0;  moved to .h file
+	int dsmout = 0;
```

Comment:
Akio's concern seems legitimate https://github.com/star-bnl/star-sw/pull/688/files#r1651407708
This new definition should not be added since it shadows a class member with the same name that was introduced about 4 years ago: https://github.com/star-bnl/star-sw/commit/4f5a8041d1f5a20c62042bce00c0c976a2765ae7?diff=unified&w=0#diff-dd89e161b0be358807922c6c11eac6634ce3fcbfbb642e31f914a88942cd28baR257

Reply:
@jml985 @tonko-lj can you follow up?

Reply:
Nobody seems to know what's going on here.   Obviously, this code is wrong because the local variable shadows the class variable.    As far as I can tell this has been true since the FCS came online.   It runs in the FCS though, and I think its Akio's code.   Akio, are you asking me to change this?   Are you asking Tonko?